packages/expo-development-client/android/src/main/java/expo/modules/developmentclient/DevelopmentClientDevMenuExtensions.java

```diff
@@ -0,0 +1,46 @@
+package expo.modules.developmentclient;
+
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Arrays;
+import java.util.List;
+
+import expo.interfaces.devmenu.DevMenuExtensionInterface;
+import expo.interfaces.devmenu.items.DevMenuAction;
+import expo.interfaces.devmenu.items.DevMenuItem;
+import expo.interfaces.devmenu.items.DevMenuItemImportance;
+import kotlin.Unit;
+
+public class DevelopmentClientDevMenuExtensions extends ReactContextBaseJavaModule implements DevMenuExtensionInterface {
+  private final ReactApplicationContext reactContext;
+
+  public DevelopmentClientDevMenuExtensions(ReactApplicationContext reactContext) {
+    super(reactContext);
+    this.reactContext = reactContext;
```

Comment:
You can access the react context from the base class using the `getReactApplicationContext` method (see https://github.com/facebook/react-native/blob/2c896d35782cd04c873aefadc947447cc30a7f60/ReactAndroid/src/main/java/com/facebook/react/bridge/ReactContextBaseJavaModule.java#L35-L39). So you don't have to save it.